Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -163,8 +197,18 @@ def fetch_indicators_from_file(args):
         if indicator_type_col_num:
             human_readable = human_readable + " indicator_type_column_number={}".format(indicator_type_col_num)
 
+    # Create indicators in demisto
+    errors = []
+    for indicator in indicator_list:
+        res = demisto.executeCommand("createNewIndicator", indicator)
+        if is_error(res[0]):
+            errors.append("Error creating indicator - {}".format(res[0]["Contents"]))
+
+        if errors:
+            return_error(json.dumps(errors, indent=4))
+
     return human_readable, {
-        "Indicator(val.Value == obj.Value && val.Type == obj.Type)": indicator_list
+        "Indicator(val.Value == obj.Value && val.Type == obj.Type)": hr_indicators_list
```

Comment:
remove the context from here, there is no need for it, it might be to big

Reply:
?

Reply:
done